src/Validations/UserValidation.tsx

```diff
@@ -19,3 +19,18 @@ export const userSchema = yup.object().shape({
         .required('Select Student or Course Creator')
         .oneOf(['student', 'courseCreator']),
 })
+
```

Comment:
Good work. I think for this courseValidation block, you can create a new file called CourseValidation.tsx and place it there.

Reply:
ok sure!

Reply:
Done!